.github/workflows/build.yml

```
@@ -5,6 +5,12 @@ on:
   release:
     types: [published]
   workflow_dispatch:
+    inputs:
+      packages:
+        description: 'list of packages to build'
```

In some other places, the description for such things is capitalized.

```suggestion
        description: 'List of packages to build'
```